### ISSUE TITLE
[BUG] "Bold tag" used on the arrow element invalidates Accessibility Test level A compliance

### ISSUE THREAD

**Issue by fleps** [select2/select2#5201]
Hello

Just FYI while doing accessibility tests using [SiteImprove Chrome extension](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc?hl) I noted that for level A compliance is needed to change any `<b>` tags to `<strong>` instead.

As Select2 adds a `<b>` tag on the arrow element it's invalidating the A test. See code from Sleect2:
```
  SingleSelection.prototype.render = function () {
    var $selection = SingleSelection.__super__.render.call(this);

    $selection.addClass('select2-selection--single');

    $selection.html(
      '<span class="select2-selection__rendered"></span>' +
      '<span class="select2-selection__arrow" role="presentation">' +
        '<b role="presentation"></b>' +
      '</span>'
    );

    return $selection;
  };
```

I tested by just changing the B to STRONG locally and everything worked so I imagine this is a simple fix.

**Comment by wolverline** [comment]
I opened a ticket for Siteimprove team but their response was just not to use it. `<b>` element isn't for expressing its original purpose here. I believe it is just a filler tag to add a glyphicon (down arrow). Maybe adding `aria-hidden="true"` is a better solution here instead of keeping the role attribute.

**Comment by fleps** [comment]
@wolverline you are correct.

Anyway, after further evaluation with a company specialized on accessibility compliance it's clear to use that this plugin is not a valid solution to customized Selects for projects that need WCAG 2.0 AA compliance, there's just too much issues on the way the plugin handles the generated wrappers and position of the dropdown, etc.

I suggest that anyone in need of an custom select with accessibility switch to some other plugin, like Bootstrap Select for example.

**Comment by eckmo** [comment]
Is this an issue that can be reopened? This is still causing failures is accessibility checkers. As @fleps suggested a few years ago, changing the tags to `<strong>` could fix the issue, but it still isn't best practice to use empty tags in general. Since the `<b>` has `display: none` applied to it, perhaps some dummy content could be added between `<strong>` tags?
